src/sass/app_2/_common/form/time_picker.scss

```diff
@@ -23,72 +22,46 @@
         &.active {
             border-color: $COLOR_BLUE;
         }
+        input {
+            text-align: center;
+        }
     }
     .time-picker-dropdown {
         font-size: 16px;
         position: absolute;
         top: 0;
-        display: flex;
-        flex-direction: column;
-        height: 0;
+        z-index: 10;
+        height: 100%;
+        max-height: 261px;
```

Comment:
Do we need `max-height`?

Reply:
Actually, I am not sure. It's something that I copied when I checked how DatePicker's animation is implemented.

Reply:
@ahkuma i'm actually changing the datepicker animation to follow dropdown animation using CSSTransition 😄, could you do the update for timepicker as well?

Reply:
@kellybinary Instead of doing it seperately, could we write a common animation and maybe put it somewhere like `animation.scss`? so we don't need 2 people to do 2 different thing that does the same thing?